bundles/org.openhab.binding.smhi/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -98,24 +134,40 @@
 		<item-type>Number:Speed</item-type>
 		<label>Wind Gust Speed</label>
 		<description>Wind gust speed</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Wind</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f m/s"/>
 	</channel-type>
 	<channel-type id="pmin">
 		<item-type>Number:Speed</item-type>
 		<label>Minimum Precipitation</label>
 		<description>Minimum precipitation intensity</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Speed</tag>
```

Comment:
```suggestion
			<tag>Rain</tag>
```

Reply:
Umm. I think we don’t have `Rain` .. but I will check..

Reply:
The channel a few rows below got Rain, so I guess we do? 🤷